doc/benchmarking_tutorial.rst

```diff
@@ -0,0 +1,97 @@
+Benchmarking tutorial
+=====================
+The [benchmarking package](https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_ros/benchmarks) provides methods to benchmark motion planning algorithms and aggregate/plot statistics.
```

Comment:
i know this is annoying, but .rst files require a different hyperlink format than markdown:

e.g. 
```
`OpenRave.org <http://openrave.org/docs/latest_stable/install/#install>`_
```